src/Compatibility/ApiDiff/Microsoft.DotNet.ApiDiff/DiffGeneratorFactory.cs

```diff
@@ -0,0 +1,125 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.DotNet.ApiSymbolExtensions;
+using Microsoft.DotNet.ApiSymbolExtensions.Logging;
+
+namespace Microsoft.DotNet.ApiDiff;
+
+public static class DiffGeneratorFactory
+{
+    /// <summary>
+    /// The default attributes to exclude from the diff.
+    /// </summary>
+    public static readonly string[] DefaultAttributesToExclude = [
+        "T:System.AttributeUsageAttribute",
+        "T:System.ComponentModel.EditorBrowsableAttribute",
+        "T:System.Diagnostics.CodeAnalysis.RequiresDynamicCodeAttribute",
+        "T:System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute"
```

Comment:
Do we really want to bake all these exclusions in?  They all seem meaningful human-authored attributes.

Reply:
These are the ones I had to exclude in the old AsmDiff already. I was asked by area owners to remove them as they weren't important for the ApiDiff and they're quite noisy.

Reply:
You're right. Removed from source code. I instead put them in a file that gets copied alongside the tool exe and I documented it in the argument description.